CHANGELOG.md

```diff
@@ -62,6 +62,11 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 #### exonum-testkit
 
+- Rollback mechanism in `Testkit` is reworked to work with checkpoints (#582):
+  - old `rollback` by blocks in `Testkit` was removed;
+  - `checkpoint` method was introduced to set checkpoints;
+  - new `rollback` rolls back to the last set checkpoint.
```

Comment:
It would be nice to add a migration path. :upside_down_face:

Reply:
Totally missed that. WIll do